README.md

```diff
@@ -78,19 +74,18 @@ Now we need to bring the Hummingbird codebase into our GOPATH. We already have
 Swift cloned in our SAIO, so it's easier for us to just symlink it into a
 folder structure within our GOPATH:
 
-    mkdir -p $GOPATH/src/github.com/openstack
-    ln -s ~/swift $GOPATH/src/github.com/openstack/swift
+    mkdir -p $GOPATH/src/github.com/troubling
+    ln -s ~/hummingbird $GOPATH/src/github.com/troubling/hummingbird
 
-Since Hummingbird work is ongoing in a feature branch, be sure you checkout the
-appropriate branch before attempting to install:
+Now, check it out:
 
-    cd $GOPATH/src/github.com/openstack/swift
-    git checkout feature/hummingbird
+    cd $GOPATH/src/github.com/troubling
+    git clone git@github.com:troubling/hummingbird.git
 
```

Comment:
I think `go get github.com/troubling/hummingbird` is sufficient. It will clone  the repo under `$GOPATH/src/github.com/troubling` automatically. One does not have to create the directory structure manually.